tests/debugging/exploration/_profiler.py

```diff
@@ -62,8 +62,8 @@ def report_func_calls(cls):
         print("")
         print(("{:<%d} {:>5}" % w).format("Function", "Calls"))
         print("=" * (w + 6))
-        for calls, func in calls:
-            print(("{:<%d} {:>5}" % w).format(func, calls))
+        for _calls, func in calls:
+            print(("{:<%d} {:>5}" % w).format(func, _calls))
```

Comment:
oh wow, I wonder who wrote this terrific line of code 🤔 
```suggestion
        for ncalls, func in calls:
            print(("{:<%d} {:>5}" % w).format(func, ncalls))
```